packages/mui-base/src/ModalUnstyled/ModalUnstyled.spec.tsx

```diff
@@ -1,9 +1,7 @@
 import * as React from 'react';
-import ModalUnstyled, {
-  ModalUnstyledRootSlotProps,
-  ModalUnstyledBackdropSlotProps,
-} from '@mui/base/ModalUnstyled';
+import ModalUnstyled from '@mui/base/ModalUnstyled';
 import { expectType } from '@mui/types';
+import { ModalUnstyledBackdropSlotProps, ModalUnstyledRootSlotProps } from './ModalUnstyled.types';
```

Comment:
Dependencies in tests should be imported as if the user of the library wrote the code - that is from the `@mui/base` package, not relative.